autogen/main/cluster.tf.tmpl

```diff
@@ -1084,6 +1085,14 @@ resource "google_container_node_pool" "windows_pools" {
       enable_secure_boot          = lookup(each.value, "enable_secure_boot", false)
       enable_integrity_monitoring = lookup(each.value, "enable_integrity_monitoring", true)
     }
+
+    dynamic "confidential_nodes" {
+      for_each = lookup(each.value, "enable_confidential_nodes", false) ? [true] : []
```

Comment:
I suggest that if `enable_confidential_nodes` is actively set to `false`, then the block be generated with enabled =  value, rather than omitted.

```suggestion
      for_each = lookup(each.value, "enable_confidential_nodes", null) != null ? [each.value.confidential_nodes] : []
```

Reply:
Done.